### PR TITLE
Move ratter to GH repo

### PR DESCRIPTION
**Pull request by BPHarris**

### CODE REVIEW

File: ratter/cli/parser.py [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+            function.
+
+            Given the regular expression <filter-string>, the results will only
+            include functions, classes, or methods whose name fully matches
+            regular expression.
+
+            Uses Python's regular expression syntax.
+        """),
+        epilog=multi_paragraph_wrap("""\
+            This is the epilogue, put a Suade thing here.
```
Comment (olliemath) [inline ratter/cli/parser.py:43]: What do you mean by this line?

**Comment by olliemath** [comment]
@brandon just a few comments on this one - although they may be superceeded by gitlab ones

File: ratter/cli/parser.py (hunk 1 above)
Comment (BPHarris) [reply ratter/cli/parser.py]: The gitlab fix (and other fixes in the mean time) have been added :)

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py3
+[testenv]
+deps=flake8
+     pytest
+     -rrequirements.txt
+commands=
+     flake8 {envsitepackagesdir}/enums
```
[the remark this answers is not in the document]
Comment (olliemath) [reply tox.ini:8]: Or if we don't use tox in CI, we can probably remove the `tox.ini`

**Comment by olliemath** [comment]
Sorry - I thought I'd added my review but it was still pending!

File: tox.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (BPHarris) [reply tox.ini]: I have removed it, I had erroneously thought it was part of PyPI packaging and copied it

**Comment by BPHarris** [comment]
@olliemath Haha, not worry, easily done :)

**Comment by BPHarris** [comment]
@olliemath Haha, easily done :) I have resolved the list thread :)

**Pull request merged**